epub33/explainers/EPUB-33-security-privacy.md

```diff
@@ -0,0 +1,103 @@
+# Security and Privacy Questionnaire for EPUB 3.3
+
+Written by Dave Cramer (invited expert) and the EPUB 3 Working Group, April 29, 2021.
+
+## Links to specifications
+
+* [EPUB 3.3 Overview](https://w3c.github.io/epub-specs/epub33/overview/), a non-normative overview of EPUB 3.3
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core/), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
+
+
+## Introduction
+
+EPUB files are presented to end users by EPUB Reading Systems, which can be constructed as web applications but are more likely to be native apps. The EPUB specifications define the file format in great detail, but generally give user agents wide latitude, subject mostly to the requirements of the embedded technologies, such as HTML, SVG, XML, and CSS. 
+
```

Comment:
I think it is worth mentioning that the majority of current reading systems are based on existing Browser Engines like Blink or WebKit. Which also means that the security and privacy features (or bugs:-) are often inherited from those engines and not inherent to the reading system itself.